apis/cache/v1alpha1/rediscluster_types.go

```diff
@@ -39,14 +30,14 @@ type RedisClusterSpec struct {
 // managed resources that are a single node, or not in cluster mode.
 // +kubebuilder:printcolumn:name="STATUS",type="string",JSONPath=".status.bindingPhase"
 // +kubebuilder:printcolumn:name="CLASS",type="string",JSONPath=".spec.classRef.name"
-// +kubebuilder:printcolumn:name="VERSION",type="string",JSONPath=".spec.engineVersion"
+// +kubebuilder:printcolumn:name="RESOURCE",type="string",JSONPath=".spec.resourceRef.name"
 // +kubebuilder:printcolumn:name="AGE",type="date",JSONPath=".metadata.creationTimestamp"
 // +kubebuilder:subresource:status
 type RedisCluster struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   RedisClusterSpec                    `json:"spec,omitempty"`
+	Spec   runtimev1alpha1.ResourceClaimSpec   `json:"spec,omitempty"`
```

Comment:
This will need to be updated in `crossplane-runtime` [here](https://github.com/crossplaneio/crossplane-runtime/blob/15eb580eecd8b4a91fe0a72cb255b6ec6a0f55cf/apis/core/v1alpha1/resource.go#L63) so that it includes a `corev1.ObjectReference` instead of `corev1.LocalObjectReference`. We should probably change name back to `ClassReference` instead of `PortableClassReference` as well.

Reply:
Also, the `Claim` interface will need to be updated [here](https://github.com/crossplaneio/crossplane-runtime/blob/15eb580eecd8b4a91fe0a72cb255b6ec6a0f55cf/pkg/resource/interfaces.go#L90) to be a `ClassReferencer` that can get and set a `corev1.ObjectReference` and angryjet will need to be updated [here](https://github.com/crossplaneio/crossplane-tools/blob/2e3995c538e39c12342474024cc0e18d047e6667/internal/method/method.go#L160) to generate the corresponding methods in `zz_generated.claim.go`.